### DISCUSSION TITLE
【深度许愿】希望支持heic格式的图片

### DISCUSSION THREAD

**Discussion by lizipeng0013** [linuxdeepin/developer-center#4233] [Features Request & Ideas | 特性请求 & 头脑风暴]
希望在文件缩略图、预览和看图中支持heic格式的图片
官方可以考虑使用libheif这个开源库，另一个图像查看软件qView也是基于libheif实现heic、heif格式支持的
还有Android系统查看heic图片，也是通过libheic支持的
libheif项目地址：https://github.com/strukturag/libheif
qView项目地址：https://github.com/jurplel/qView

**Comment by BLumia** [comment]
https://github.com/orgs/linuxdeepin/discussions/2913
